egs2/qasr_tts/tts1/conf/tuning/train_fastspeech2.yaml

```diff
@@ -0,0 +1,102 @@
+# This configuration is for ESPnet2 to train FastSpeech2.
```

Comment:
It would be great if you can simply the configuration used.

Reply:
Have you tested all the corresponding configurations? If not, I would suggest to just keep the one you have tested.